### DISCUSSION TITLE
http and https s3 bucket

### DISCUSSION THREAD

**Discussion by yelhouti** [minio/minio#12350] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I would like to connect to an s3 bucket locally without ssl and remotly with ssl.

**Describe the solution you'd like**
is it possible to have two ports one foreach, so one can be exposed to outside the cluster and one only to inside the cluster.

**Comment by klauspost** [comment, marked as answer]
No, that is not possible and would be a huge change we are unlikely to undertake. You can of course investigate options to proxy your cluster.

Seems like the [træffik cookbook](https://docs.min.io/docs/how-to-run-multiple-minio-servers-with-traef-k.html) contains an example with that. Maybe that can help.
